docs/cluster/networking.md

```diff
@@ -0,0 +1,142 @@
+# Networking in Kubernetes
+
+## Docker model
+
+Before discussing the Kubernetes approach to networking, it is worthwhile to
+review the "normal" way that networking works with Docker.  By default, Docker
+uses host-private networking.  It creates a virtual bridge, called `docker0` by
+default, and allocates an [RFC1918](https://tools.ietf.org/html/rfc1918) range
```

Comment:
RFC1918 does not use the word "range" anywhere. 

For precision, how about " ... allocates a subnet from one of the private address blocks defined in RFC1918".